xm-commons-timeline/src/main/java/com/icthh/xm/commons/timeline/config/TimelineInterceptor.java

```diff
@@ -97,7 +106,38 @@ private boolean isIgnoredRequest(HttpServletRequest request) {
                 }
             }
         }
-        return false;
+
+        String responseBody = getResponseContent(response);
+        String typeKey = getEntityField(responseBody, TYPE_KEY);
+
+        return isNotEmpty(ignoredTypeKeys) && ignoredTypeKeys.contains(typeKey);
+    }
+
+    private String getResponseContent(HttpServletResponse response) {
+        if (response instanceof ContentCachingResponseWrapper) {
+            return new String(((ContentCachingResponseWrapper) response).getContentAsByteArray());
+        }
+        if (response instanceof HttpServletResponseWrapper
+            && ((HttpServletResponseWrapper) response).getResponse() instanceof ContentCachingResponseWrapper) {
+            return new String(((ContentCachingResponseWrapper) ((HttpServletResponseWrapper) response)
+                .getResponse()).getContentAsByteArray());
+        }
+        log.warn("Empty response content because of unsupported response class {}", response.getClass());
+        return "";
+    }
+
+    public String getEntityField(String entity, String field) {
```

Comment:
This method is copy-paste and duplicated with TimelineEventProducer class. What do you think to extract this method to some utility class and make static?